workers/ohsome_quality_analyst/api/api.py

```diff
@@ -308,3 +312,18 @@ def _remove_svg_from_properties(properties: dict) -> None:
     elif isinstance(geojson_object, FeatureCollection):
         for feature in geojson_object["features"]:
             _remove_svg_from_properties(feature["properties"])
+
+
+def remove_html_from_properties(
+    geojson_object: Union[Feature, FeatureCollection]
+) -> None:
+    def _remove_html_from_properties(properties: dict) -> None:
+        for key in list(properties.keys()):
+            if fnmatch.fnmatch(key, "*result.html"):
+                del properties[key]
+
+    if isinstance(geojson_object, Feature):
+        _remove_html_from_properties(geojson_object["properties"])
+    elif isinstance(geojson_object, FeatureCollection):
+        for feature in geojson_object["features"]:
+            _remove_html_from_properties(feature["properties"])
```

Comment:
This is exactly the same as `remove_svg_from_properties`, isn't it? I would suggest to use a more generic function named like `remove_from_properties(geojson_object, key)` then called like `remove_from_properties(geojson_object, "*result.html")`. This would prevent code duplication.

Reply:
Adressed in #280